src/test/java/com/project/group4/models/json/DesignatedProductJsonTest.java

```diff
@@ -0,0 +1,22 @@
+package com.project.group4.models.json;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DesignatedProductJsonTest {
+
+    private String classCode;
+    private String className;
+
+    @Test
+    public void buildTest() {
+        DesignatedProductJson designatedProductJson = DesignatedProductJson.builder()
+                .classCode(this.classCode)
```

Comment:
this.classCode, this.className เอาค่ามาจากไหน แล้ว assertEqual คือ equal null ? กลับไปเช็คใหม่นะครับ ถ้ายังงี้ มันเป็น failure test case นะ